### ISSUE TITLE
Possibility to pick number of samples for calls to posterior predictive?

### ISSUE THREAD

**Issue by AlexanderFengler** [bambinos/bambi#739]
It is often useful / sufficient for posterior predictives (examples for purposes of plotting) to be based on less than all samples from a given call to `.fit()`. 

It seems like currently one cannot pick the number of samples at the level of the posterior predictive. 

Would it be possible to add this as a small feature (or allow passing through of such parameter to the `pm.sample_posterior_predictive()` call?

**Comment by tomicapretto** [comment]
It's definitely possible but not 100% trivial. Bambi does not use `pm.sample_posterior_predictive()` under the hood (maybe we should!). It uses `pm.draw(dist(**kwargs))` and the number of draws we obtain depends on the shape of the array(s) in `kwargs`. To decrease the number of draws from the posterior predictive distribution we should first downsample the posterior draws. I'll check what PyMC does in these cases. Maybe the answer is that we should move to `pm.sample_posterior_predictive()`.

**Comment by digicosmos86** [comment]
@tomicapretto We implemented a temporary solution which just samples the `posterior` group of the trace object and then pass it to `model.predict`. Do you foresee issues with this?

**Comment by tomicapretto** [comment]
> @tomicapretto We implemented a temporary solution which just samples the `posterior` group of the trace object and then pass it to `model.predict`. Do you foresee issues with this?

Nope, I think we could use it in Bambi. Do you have it somewhere? Feel free to open a PR, or I can open one myself. Thanks!

**Comment by digicosmos86** [comment]
@tomicapretto Sure! I'll do a PR soon. Thanks!

**Comment by tanishy7777** [comment]
can I work on this? Could you elaborate what to do? @tomicapretto

**Comment by tomicapretto** [comment]
We want to modify [this method](https://bambinos.github.io/bambi/api/Model.html#bambi.Model.predict) to allow users to pass the number of draws when we obtain draws from the posterior predictive distribution (i.e. when `kind="response"`). Right now, it uses as many draws as draws from the posterior. Users may want to use fewer draws.

**Comment by tomicapretto** [comment]
I think this is simple now that we use `pm.sample_posterior_predictive` under the hood. 
Look at https://www.pymc.io/projects/docs/en/stable/api/generated/pymc.sample_posterior_predictive.html#controlling-the-number-of-samples
